internal/logging/logging.go

```diff
@@ -237,6 +237,26 @@ func ReadTail() string {
 	return logTail.Read()
 }
 
+var svcTailProvider func() string
+
+// SetSvcTailProvider registers the given function as providing the state-svc log tail.
+// Fetching that log via pkg/platform/model is not possible here due to an import cycle, so the
+// given function is expected to do that.
+func SetSvcTailProvider(f func() string) {
+	svcTailProvider = f
+}
+
+// ReadSvcTail returns the tail end of the state-svc log, just like ReadTail().
+// This exists in order to have Rollbar send both state logs and state-svc logs together.
+// If no provider function was given earlier, this function returns the empty string, so calling it
+// from state-svc is safe. (i.e. state-svc is sending its own error log to Rollbar.)
+func ReadSvcTail() string {
+	if svcTailProvider != nil {
+		return svcTailProvider()
+	}
+	return ""
```

Comment:
Please consider returning something like "no svc log tail provider set" for the sake of debugging.

Reply:
Okay, I will use a named constant here.